### PR TITLE
Fix docker image build

### LINKED ISSUE #4: Add swagger

### PR DESCRIPTION
**Pull request by Uxio0**
- Remove not needed dependencies
- Add gunicorn missing configuration
- Organize .env samples
- Update README.md
- Make sure swagger works
- Closes #4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,16 @@ Returns fiat prices for base currencies and ERC20 tokens.
...
 
-Environment variables:
+```bash
+cp .env.sample .env
+# You can configure variables in .env
+docker compose build
+docker compose up
+```
+
+Then go to http://localhost:8000
```
Comment (moisses89) [inline README.md:20]: I'm thinking that we are mixing configuration and execution here, could we separate them in different sections?

File: docker/web/Dockerfile [hunk 2]
```diff
@@ -38,4 +38,4 @@ COPY --chown=python:python . .
 # Use numeric ids so kubernetes identifies the user correctly
 USER 999:999
 
-RUN DJANGO_SETTINGS_MODULE=config.settings.production DJANGO_DOT_ENV_FILE=.env.tracing.sample python manage.py collectstatic --noinput
+RUN DJANGO_SETTINGS_MODULE=config.settings.production DJANGO_DOT_ENV_FILE=.env.sample python manage.py collectstatic --noinput
```
Comment (moisses89) [inline docker/web/Dockerfile:41]: I suppose that here should be .env instead .env.sample.

File: README.md (hunk 1 above)
Comment (Uxio0) [reply README.md:20]: You are right

File: docker/web/Dockerfile (hunk 2 above)
Comment (Uxio0) [reply docker/web/Dockerfile:41]: No, as this is only a placeholder as it needs a file to exist, so it doesn't matter which one

**Review by moisses89 — APPROVED** [review]

**Pull request merged**